nuxt.config.js

```diff
@@ -1,61 +1,104 @@
 module.exports = {
   head: {
-    title: 'Vue Vixens - Workshops for Foxy People to Learn Vue.js',
+    title: "Vue Vixens - Workshops for Foxy People to Learn Vue.js",
```

Comment:
**nitpick**: could you please revert this (and all upcoming) changes back to single quotes?

Reply:
single quotes are go!